examples/cpy/axpb_jit.py

```diff
@@ -0,0 +1,22 @@
+from pysph.cpy.api import Elementwise, wrap, get_config
+from pysph.cpy.jit import annotate
```

Comment:
I don't really like that you are defining another analogous annotate, there should be one `annotate` decorator from api and not one from jit and another from types.  This will make it hard for users and developers.  Basically if there are arguments, it will treat it as the old annotate and if there are none, it jits.  So basically merge the two into one.